### PR TITLE
infra - break storybook build if a stylable warning is thrown

### PR DESCRIPTION
**Pull request by shlomitc**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6406

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -9,8 +10,14 @@ const testkitsWarning = `
 To learn how to initialize and use testkits, see <a href="/?path=/story/introduction-testing--testing" target="_blank">Testing guide</a>
 `;
 
+const patchStylablePlugin = (config) => {
+  config.plugins.pop() // Remove the yoshi built-in stylable plugin
```
Comment (EgoziE) [inline .storybook/webpack.config.js:14]: Can we somehow make sure the one popped out is the one we need?

Comment (shlomitc) [reply .storybook/webpack.config.js:14]: not sure there's a way...
This is how the storybook webpack config looks like:
`config.plugins = [...(config.plugins || []), new StylableWebpackPlugin()];`

Comment (EgoziE) [reply .storybook/webpack.config.js:14]: Low chances it will break, but in case it does we'll know who to go to 😜

**Pull request merged**